Makefile

```diff
@@ -85,11 +81,19 @@ third-party-try-opt: third-party-try-re2 third-party-try-gmp
 third-party-try-re2: FORCE
 	-@if [ -z "$$CHPL_REGEXP" ]; then \
 	cd third-party && $(MAKE) try-re2; \
+	if [ "llvm" = `${CHPL_MAKE_HOME}/util/chplenv/chpl_llvm.py` ]; then \
+	export CHPL_TARGET_COMPILER=clang-included && \
+	$(MAKE) try-re2; \
+	fi \
 	fi
 
 third-party-try-gmp: FORCE
 	-@if [ -z "$$CHPL_GMP" ]; then \
 	cd third-party && $(MAKE) try-gmp; \
+	if [ "llvm" = `${CHPL_MAKE_HOME}/util/chplenv/chpl_llvm.py` ]; then \
```

Comment:
Hmm, that's an interesting question. Right now clang-included means use the clang in third-party/llvm/install/... If we supported a system llvm, e.g. CHPL_LLVM=system, I'm not sure what would mean w.r.t. the target compiler. Let's discuss with Michael outside this PR.